### ISSUE TITLE
now doesnt recognize vlc.VideoHandler. Latest Commit

### ISSUE THREAD

**Issue by TheRealJake12** [polybiusproxy/hxCodec#140]
For some reason, I get an error saying that it can't find the ``vlc.VideoHandler;``. whats going on?

**Comment by TheRealJake12** [comment]
![image](https://user-images.githubusercontent.com/84357907/174508269-effd99b2-9644-4b9f-9aec-5ede76f075bb.png)

**Comment by TheRealJake12** [comment]
looked at the code again. ``src.VideoHandler`` also does not work

**Comment by jigsaw-4277821** [comment]
Put simple `import VideoHandler;`

**Comment by jigsaw-4277821** [comment]
You don't need to put another

**Comment by TheRealJake12** [comment]
I did nothing worked

**Comment by TheRealJake12** [comment]
![image](https://user-images.githubusercontent.com/84357907/174533949-14e27716-d372-4931-aa68-81cef652f493.png)
![image](https://user-images.githubusercontent.com/84357907/174533972-522a86b1-ae1d-4a65-b9fb-e2c9bbf62776.png)
1

**Comment by jigsaw-4277821** [comment]
Don't import it then and be sure you updated the library

**Comment by TheRealJake12** [comment]
I've reinstalled it multiple times

**Comment by TheRealJake12** [comment]
and not importing still says it cant find VideoHandler

**Comment by polybiusproxy** [comment]
Have you tried compiling? VS Code sometimes goes crazy with files inside packages.

**Comment by TheRealJake12** [comment]
![image](https://user-images.githubusercontent.com/84357907/174591285-86a85268-8541-49c4-8420-1bea3d8db0cc.png)

**Comment by TheRealJake12** [comment]
(appveyor)

**Comment by jigsaw-4277821** [comment]
> ![image](https://user-images.githubusercontent.com/84357907/174591285-86a85268-8541-49c4-8420-1bea3d8db0cc.png)

Because isn't in VLC folder anymore

**Comment by TheRealJake12** [comment]
yeah but doing src.VideoHandler doesnt work either. And just VideoHandler leaves it confused

**Comment by jigsaw-4277821** [comment]
You don't need to import it

**Comment by TheRealJake12** [comment]
but not importing it gives errors

**Comment by TheRealJake12** [comment]
![image](https://user-images.githubusercontent.com/84357907/174931457-d84f7566-1e25-4412-803b-e623161b8c1f.png)
this is without importing

**Comment by TheRealJake12** [comment]
just revert the move
easy fix
